composer.json

```diff
@@ -31,7 +31,8 @@
     "autoload": {
         "psr-4": {
             "AsyncAws\\": "src"
-        }
+        },
+        "files": ["src/Core/Test/PHPUnitTestCase.php"]
```

Comment:
Hmmm.. I'm not sure it should be loaded for non-dev

Reply:
In already is.. And it is just a small file.

Reply:
It needs to be here, or it will not be auto loaded from ie the s3 client.

Reply:
I could maybe improve

Reply:
S3 client don't need it's unless they use it in tests, and in that case phpunit should be autoloaded